sources/torch.optim/configen.yaml

```diff
@@ -1,9 +1,13 @@
+# Copyright (c) 2020, Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
Regarding the change from `conf` directory to `sources` directory, I think this is a good idea. I think the term conf/config/configen is very overloaded in this setting. (something I commented on a while back too). It made knowing where to look for things imo overly difficult until I got used to the lay of the land. I might even argue this should be propagated to the default dir for configen.